File: app/src/main/java/com/numerad/quickseries/json/SocialMedia.java
package com.numerad.quickseries.json;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import javax.annotation.Generated;

@Generated("com.robohorse.robopojogenerator")
public class SocialMedia{

	@SerializedName("twitter")
	private List<String> twitter;

	@SerializedName("youtubeChannel")
	private List<String> youtubeChannel;

	@SerializedName("facebook")
	private List<String> facebook;

	public void setTwitter(List<String> twitter){
		this.twitter = twitter;
	}

	public List<String> getTwitter(){
		return twitter;
	}

	public void setYoutubeChannel(List<String> youtubeChannel){
		this.youtubeChannel = youtubeChannel;
	}

	public List<String> getYoutubeChannel(){
		return youtubeChannel;
	}

	public void setFacebook(List<String> facebook){
		this.facebook = facebook;
	}

	public List<String> getFacebook(){
		return facebook;
	}

	@Override
 	public String toString(){
		return 
			"SocialMedia{" + 
			"twitter = '" + twitter + '\'' + 
			",youtubeChannel = '" + youtubeChannel + '\'' + 
			",facebook = '" + facebook + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/mohamadk/quickseries/pages/gride/items/places/SocialMedia.kt
package com.mohamadk.quickseries.pages.gride.items.places

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class SocialMedia (
    @SerializedName("youtubeChannel")
    val youtubeChannel:List<String>,
    @SerializedName("twitter")
    val twitter:List<String>,
    @SerializedName("facebook")
    val facebook:List<String>
):Parcelable {

}
